mapper/dm_agent/README.md

```diff
@@ -0,0 +1,31 @@
+# Device Monitoring
+
+Device monitoring is enabled on `thin-edge.io` using `collectd` and the `tedge-dm-agent` component.
+
+# Prerequisites
+
+Install collectd if not already installed: 
+
+`sudo apt install collectd`
```

Comment:
Is there any version requirement for `collectd`? Maybe it would be good to have short `requirements` section?

Reply:
`collectd-core` is enough. The difference is 35.2 MB of disk space.